tests/contract_testcases/polkadot/functions/external_call_args.sol

```diff
@@ -0,0 +1,71 @@
+interface IUniswapV2Router01 {
+    function WETH() external pure returns (address);
+
+    function swapExactETHForTokens(
+        uint amountOutMin,
+        address[] calldata path,
+        address to,
+        uint deadline
+    ) external payable returns (uint[] memory amounts);
+
+    function swapTokensForExactETH(
+        uint amountOut,
+        uint amountInMax,
+        address[] calldata path,
+        address to,
+        uint deadline
+    ) external returns (uint[] memory amounts);
+
+    function swapExactTokensForETH(
+        uint amountIn,
+        uint amountOutMin,
+        address[] calldata path,
+        address to,
+        uint deadline
+    ) external returns (uint[] memory amounts);
+
+    function swapETHForExactTokens(
+        uint amountOut,
+        address[] calldata path,
+        address to,
+        uint deadline
+    ) external payable returns (uint[] memory amounts);
+}
+
+library SafeMath {
+    function mul(uint256 a, uint256 b) internal pure returns (uint256) {
+        if (a == 0) return 0;
+        uint256 c = a * b;
+        require(c / a == b, "SafeMath: multiplication overflow");
+        return c;
+    }
+
+    function div(uint256 a, uint256 b) internal pure returns (uint256) {
+        require(b > 0, "SafeMath: division by zero");
+        return a / b;
+    }
+}
+
+contract Swap_ETH_TO_USDX {
+    using SafeMath for uint;
+    address distTokens;
+    uint deadline;
+    uint feerate;
+    IUniswapV2Router01 public uniswapRouter;
+
+    receive() external payable {
+        address[] memory paths = new address[](2);
+        paths[0] = uniswapRouter.WETH();
+        paths[1] = distTokens;
+        uint[] memory amounts = uniswapRouter.swapExactETHForTokens{
+            value: msg.value.mul(feerate).div(10000)
+        }({
+            amountOutMin: 0,
+            path: paths,
+            to: address(this),
+            deadline: block.timestamp + deadline
+        });
+    }
+}
+// ---- Expect: diagnostics ----
+// warning: 61:20-53: conversion truncates uint256 to uint128, as value is type uint128 on target polkadot
```

Comment:
`polkadot` is written here with a lower case p, while it is written with an upper case P elsewhere. Please, use a single form.

Reply:
We often do `format!("you cannot do foo on target {}", ns.target);` The `fmt::Display` for `Target` prints lower case, and I don't know why. I think this can be changed to start with an upper case letter, it is Polkadot and Solana after all, not polkadot and solana.

Reply:
>The `fmt::Display` for `Target` prints lower case, and I don't know why.

It is written with a lower case.
https://github.com/hyperledger/solang/blob/f9230fc5ab2d957e90153fdfaaa5701f241f6add/src/lib.rs#L39-L41

Reply:
Yes, that's the code I am referring to.

Reply:
I think what Sean meant was that he doesn't know why it's implemented like that. I'm happy to change capitalization  in a follow up PR, but not in this one.